### PR TITLE
My Entry to the Challenge

### PR DESCRIPTION
**Pull request by naclcaleb**
My program can be found [here](./3-conditionals-and-logic/challenge/HelloWorldInDifferentProgrammingLanguages.cpp).

I do need to ask about some qualifications.
This program uses a few things that are not in the course yet (like a function and the `std::string` type), and also a loop that is talked about in the next lesson.

Is that okay, or should I try and take those elements out? I could do it, but the code would be much messier.

Secondly, I have a `switch` statement instead of `if/elseif/else`, and I can change that too if you'd like.
I do actually (rather by coincidence) have an `if/else` statement near the end also.

### CODE REVIEW

**Comment by naclcaleb** [comment]
Sorry, I realize my link didn't work. Let me try again...

**Comment by naclcaleb** [comment]
Ok, my code is here: https://github.com/naclcaleb/Codecademy-Learn-C-Plus-Plus/blob/master/3-conditionals-and-logic/challenge/HelloWorldInDifferentProgrammingLanguages.cpp

I also made a separate Github repo for it [here](https://github.com/naclcaleb/ConditionalsAndLogicChallenge), with a separate branch containing a version in which I do not use the aforementioned techniques.

**Comment by sonnynomnom** [comment]
@naclcaleb amazing! what's ur twitter? so i can tag you when we announce the winners

**Comment by naclcaleb** [comment]
I don't have Twitter, sorry! That's why I had to use the GitHub pull request.

My best alternative would just be by Github profile.

**Comment by naclcaleb** [comment]
Yea, my GitHub is probably the best option.

I'm kind of young, so I don't have Twitter as of right now.

**Comment by naclcaleb** [comment]
I also do not have a YouTube channel, so I am unable to comment during the livestreams. Just so you know.

**Comment by naclcaleb** [comment]
If you need a Twitter profile, I could provide on of my parents', but otherwise I would just use the GitHub profile.

**Comment by sonnynomnom** [comment]
@naclcaleb yes your parents' twitter is perfect!

**Comment by naclcaleb** [comment]
Ok, here it is: @lisa_hester

**Comment by sonnynomnom** [comment]
it looks like she disabled messaging. so let's try this for now: https://get.printfection.com/xtwmd/9097001397/gwdyjxhtyb

**Comment by sonnynomnom** [comment]
and congrats!

**Comment by naclcaleb** [comment]
Thanks! It looks like that will work.
